loaders/src/glTF/2.0/babylon.glTFLoader.ts

```diff
@@ -600,63 +600,67 @@ module BABYLON.GLTF2 {
             if (primitive.targets !== undefined) {
                 morphTargetManager = new BABYLON.MorphTargetManager();

```

Comment:
Good point. With the additional checks before assigning it to the mesh, I can move the object creation out a level.